packages/animation/package.json

```diff
@@ -26,7 +26,9 @@
   "exports": {
     "default": "./src/index.js"
   },
-  "main": "./src/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist-module/index.js",
```

Comment:
Maybe we need to remove `./` or else the imports would not work in another library. That's what we have in the rollup example repo
https://github.com/rollup/rollup-starter-lib/blob/88a2d55d449e2cb12191199d993c08bc07c90be2/package.json#L4-L5

and in Gutenberg 
https://github.com/WordPress/gutenberg/blob/262d4675532ead489a37bfb7957a84f9b3bd8817/packages/block-library/package.json#L24-L25